extensions/ApiExplorer/SpecialApiExplorer.php

```diff
@@ -50,16 +50,15 @@ public function execute( $par = null ) {
 		wfProfileIn( __METHOD__ );
 
 		// TODO: Make this work for ResourceLoader (Wikia isn't using RL yet at the time of this writing).
-		// Wikia has the cachebuster in the wgExtensionPath (we rewrite that in varnish because many proxies won't cache things that have "?" in the URL), but other MediaWikis need the style-version in the querystring.
-		$cbSuffix = ( isset( $wgCityId ) ? "?{$wgStyleVersion}" : "" );
-		$wgOut->addScript( "<script type=\"text/javascript\" src=\"{$wgExtensionsPath}/wikia/JavascriptAPI/Mediawiki.js{$cbSuffix}\"></script>" );
-		$wgOut->addScript( "<script type=\"text/javascript\" src=\"{$wgExtensionsPath}/ApiExplorer/apiExplorer.js{$cbSuffix}\"></script>" );
-		$wgOut->addScript( "<link rel=\"stylesheet\" type=\"text/css\" href=\"{$wgExtensionsPath}/ApiExplorer/apiExplorer.css{$cbSuffix}\" />" );
+		// Wikia interpolates wgStyleVersion in the wgExtensionPath (we rewrite that in varnish because many proxies won't cache things that have "?" in the URL), but other MediaWikis need the style-version in the querystring.
+		$wgOut->addScript( "<script type=\"text/javascript\" src=\"{$wgExtensionsPath}/wikia/JavascriptAPI/Mediawiki.js{$wgStyleVersion}\"></script>" );
+		$wgOut->addScript( "<script type=\"text/javascript\" src=\"{$wgExtensionsPath}/ApiExplorer/apiExplorer.js{$wgStyleVersion}\"></script>" );
+		$wgOut->addScript( "<link rel=\"stylesheet\" type=\"text/css\" href=\"{$wgExtensionsPath}/ApiExplorer/apiExplorer.css{$wgStyleVersion}\" />" );
 
 		ob_start();
 			$buttonHeight = 15;
-			$collapseSrc = "$wgExtensionsPath/ApiExplorer/collapse.png$cbSuffix";
-			$expandSrc = "$wgExtensionsPath/ApiExplorer/collapse.png$cbSuffix";
+			$collapseSrc = "$wgExtensionsPath/ApiExplorer/collapse.png$wgStyleVersion";
+			$expandSrc = "$wgExtensionsPath/ApiExplorer/collapse.png$wgStyleVersion";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$expandSrc" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app